.github/workflows/release-desktop-app.yml

```diff
@@ -45,6 +45,10 @@ jobs:
       NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID: ${{ secrets.NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID }}
       AFFINE_GOOGLE_CLIENT_ID: ${{ secrets.AFFINE_GOOGLE_CLIENT_ID }}
       AFFINE_GOOGLE_CLIENT_SECRET: ${{ secrets.AFFINE_GOOGLE_CLIENT_SECRET }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
```

Comment:
Should also add these to `make-windows` as well

Reply:
you re correct